packages/storybook/package.json

```diff
@@ -19,6 +19,7 @@
     "@storybook/polymer": "^5.3.13",
     "@storybook/storybook-deployer": "^2.8.1",
     "@vaadin/vaadin-button": "^2.1.5",
+    "@vaadin/vaadin-context-menu": "^4.5.0",
```

Comment:
You don't need this as separate dependency

* it's already imported in @conversionxl/cxl-ui
* even if not, I don't see `import @vaadin/vaadin-context-menu` anywhere in this PR story set, which means the dependency should be something else.